### DISCUSSION TITLE
Guys, how to add news itens (sets)

### DISCUSSION THREAD

**Discussion by gustavofox** [flareteam/flare-game#958] [General]
Thanks for any help...

**Comment by dorkster** [comment]
Have a look at `items/sets.txt`: https://github.com/flareteam/flare-game/blob/master/mods/empyrean_campaign/items/sets.txt

It should be mostly self-explanatory. The important properties are `items` and `bonus`. `items` is a list of the item IDs (from `items/items.txt`) that belong to the set. The `bonus` can be broken down like so: `bonus=<required number of equipped items>,<type of bonus>,<value of bonus>`
